src/Console/Output/ProcessCommandReporter.php

```diff
@@ -28,18 +28,23 @@ public function reportLoadedRectors(): void
         $this->symfonyStyle->title(sprintf(
             '%d Loaded Rector%s',
             $this->rectorCollector->getRectorCount(),
-            $this->rectorCollector->getRectorCount() === 1 ? '' : 's'
+            $this->rectorCollector->getRectorCount() === 1 ?: 's'
         ));
 
         $rectorList = $this->sortByClassName($this->rectorCollector->getRectors());
-        foreach ($rectorList as $rector) {
-            $this->symfonyStyle->writeln(sprintf(
-                ' - %s',
-                get_class($rector)
-            ));
-        }
-
-        $this->symfonyStyle->newLine();
+
+        $this->symfonyStyle->listing(array_map(function ($rector): string {
```

Comment:
Missing typehint on `function (Rector $rector)`

Reply:
We definitely need to find a Sniff for our closures :-1: